scripts/mirror-repos.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -eE
+
+mydir=$(dirname $0)
+topdir=$mydir/..
+
+. "$mydir/lib/misc.sh"
+
+[ $# = 1 ] || die "need exactly 1 non-option argument"
+DIST="$1"
+maybe_set_srcurl "$DIST"
+
+command -v lftp >/dev/null || die "required tool not found: lftp"
+
+lftp -c mirror \
+     --verbose \
+     --delete \
+     --exclude="/Source/|-debuginfo-|-devel[-_]" \
+     $SRCURL ~/mirrors/xcpng/$DIST
```

Comment:
This path should be configurable. For example, I wouldn't put mirrors in this directory on my own system.

Reply:
right